public/stylesheets/common.css

```diff
@@ -1271,15 +1271,20 @@ body #themepicker div.is3d,
   background: #f0f0f0;
 }
 .is2d #themepicker div.color_demo {
-  width: 64px;
-  height: 32px;
+  width: 66px;
+  height: 34px;
   background-size: 256px;
   border: 1px solid #d0d0d0;
+  box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  -ms-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
```

Comment:
Thanks. According to http://caniuse.com/#search=box-sizing, vendor prefixes are quite superfluous, I'll remove them.